### PR TITLE
Currently buildah run is not handling command options correctly

### PR DESCRIPTION
**Pull request by rhatdan**
This patch will allow commands like

buildah run $ctr ls -lZ /

To work correctly.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
LGTM

File: cmd/buildah/run.go [hunk 1]
```diff
@@ -43,12 +43,14 @@ var (
...
-		Flags:       runFlags,
-		Action:      runCmd,
-		ArgsUsage:   "CONTAINER-NAME-OR-ID COMMAND [ARGS [...]]",
+		Name:                   "run",
+		Usage:                  "Run a command inside of the container",
+		Description:            runDescription,
+		Flags:                  runFlags,
+		Action:                 runCmd,
+		ArgsUsage:              "CONTAINER-NAME-OR-ID COMMAND [ARGS [...]]",
+		SkipArgReorder:         true,
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/run.go]: should we add these to the other commands?

Comment (rhatdan) [reply cmd/buildah/run.go]: THe skip arg reorder is just for executing commands after the image. So not sure for it.  But the supports short names might be usefull for buildah from/buid, if they support single letter options.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Comment by umohnani8** [comment]
"Error determining manifest MIME type for docker://registry.fedoraproject.org/fedora-minimal:latest: unauthorized: authentication required" -  fedoraproject.org is down?

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by cgwalters** [comment]
FWIW I was previously doing:

`buildah config ${container} --entrypoint blah` etc., and that broke after this PR.

But eh, after reading now I understand why the change was made.

**Comment by cgwalters** [comment]
(It's tricky since the rest of my code is lots of `buildah run ${container} arguments...`)

**Pull request closed without merging**